### ISSUE TITLE
Empty "Available at these locations" heading within a service accordion

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17165]
## Description
In a couple places, I have found service accordions with an empty heading for "Available at these locations", meaning the heading is displaying but no content underneath it:
![Screenshot 2024-02-08 at 9 00 55 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/ff297153-7ae9-4542-a2d8-fe84fea02b6d)

This is an issue for accessibility and also within the code, there is an empty `<ul>` element which poses a second issue for accessibility:
![Screenshot 2024-02-08 at 9 02 21 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/6bb31813-3252-455b-95f6-c9891d961d7e)

Is this showing because the template was build to assume if a service was added it would be available at at least 1 location? If that's true, is this an exception and we would need to edit the template to make it work for these exceptions or is this simply editor error and they need to add the locations here?

Counter example: https://www.va.gov/alaska-health-care/health-services/#item-social-work
Alaska Social Work (system) has no listed Facilities but also no H3 / empty ul. 

### Instances
- [Alaska VAMC health services page](https://www.va.gov/alaska-health-care/health-services/), the last service accordion
- [Eastern Oklahoma Health Care](https://www.va.gov/eastern-oklahoma-health-care/health-services/#item-suicide-prevention) Suicide Prevention

## Implementation note
Add logic to check: only add H3 / ul if there are items in the list. 

## Acceptance Criteria
- [ ] H3 / ul for Available at these locations should only show if, in fact, the service is available at location. Verified on provided example pages: 
  - [ ] https://www.va.gov/alaska-health-care/health-services/#item-social-work
  - [ ] https://www.va.gov/eastern-oklahoma-health-care/health-services/#item-suicide-prevention
- [ ] Requires accessibility review

**Comment by xiongjaneg** [comment]
@laflannery Michelle recommended approaching this on both fronts, technically by removing this in the case where there is nothing listed and editorially by having VHA DM communicate to editors which required services are missing. As you find more of these, please let us know and we'll add to VHA DM sync. Adding these to next sync.

**Comment by mmiddaugh** [comment]
@laflannery can you give this a defect level to assist with prioritization, please?

**Comment by mmiddaugh** [comment]
I believe this issue occurs when the editor has added a service at the system level but fails to add the facilities at which it is available. This translates directly into a disconnect in the data which prevents locations from being discoverable in the Facility Locator and/or other applications in the ecosystem. We can handle this from an editor education perspective.

**Comment by Agile6MSkinner** [comment]
@mmiddaugh Sounds like we don't want to do any engineering work here then? What channel can we use for editor education so that we can close this?

**Comment by Agile6MSkinner** [comment]
Add to VHA training agenda, close ticket
